### DISCUSSION TITLE
Feature Request : Configuration Export / Import Function

### DISCUSSION THREAD

**Discussion by Noschvie** [linuxserver/Heimdall#675] [Ideas]
It would be fine to have a function for exporting the configured items as json file and vice versa.

**Comment by wstrater** [comment]
I second the request. Hope the custom images get added to the export/import. They could be Base64 encoded.

Wes.

**Comment by asmusz-adobe** [comment]
I third the request!  :-)

**Comment by bob454522** [comment]
I wish it were more clear if the current import / export buttons are MERGE (ie the import)  or if it overwrites.

Ie if I manually want to add 6 new items, can I import those in a custom made JSON (based on the format of an export), or should I export, add the new items and import the entirety of all items (or willl i end up with duplcates in that case)?
  (I can't find any documentation or info on this, and also if the icons are eliminated during an import export this should be noted in the GUI)
tks
